march_rqt_note_taker/src/march_rqt_note_taker/filter_map.py

```diff
@@ -0,0 +1,47 @@
+from rosgraph_msgs.msg import Log
+
+
+class FilterMap:
```

Comment:
I am not sure about this, but something to consider: Shouldn't we also have an option to add filters that exclude messages even when they are accepted by other filters.

Reply:
I had also thought about this, but thought it would not be necessary. If others also think this would be useful I could implement it.